crates/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -260,6 +260,15 @@ impl AcirContext {
         }
     }
 
+    // Return Some(constant) if the given AcirVar refers to a constant value,
+    // otherwise return None
+    pub(crate) fn get_constant(&self, var: &AcirVar) -> Option<FieldElement> {
+        match self.vars[var] {
+            AcirVarData::Const(field) => Some(field),
+            _ => None,
+        }
+    }
+
```

Comment:
If we make `var_to_expression` be `pub(crate)` then we can replace calls to this with `self.var_to_expression(var).to_const()`.

In any case, we want to handle undefined acir variables in the same way between both methods.

Reply:
I removed usage of this method and switched to your suggestion